src/screens/PairDevices/index.js

```diff
@@ -204,8 +229,17 @@ function PairDevicesInner({ navigation, route }: Props) {
 
   const onDone = useCallback(
     (device: Device) => {
-      navigation.goBack();
-      route.params?.onDone?.(device);
+      // To avoid passing a onDone function param that is not serializable
+      if (route.params?.onDoneNavigateTo === ScreenName.SyncOnboardingWelcome) {
+        console.log("PairDevices: 🦮 navigate directly to SyncOnboarding");
```

Comment:
It's to avoid passing a `onDone` function as a route argument. It should be avoided according to [the doc of react-navigation](https://reactnavigation.org/docs/troubleshooting/#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state). To be sure that the navigation flow works correctly, we should only pass serialisable params